test/integration/targets/vmware_inventory/runme.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+[[ -n "$DEBUG" || -n "$ANSIBLE_DEBUG" ]] && set -x
+
+set -euo pipefail
+
+# Get Current test directory
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+contrib_dir=$(pwd)/../../../../contrib/inventory
+cd "${contrib_dir}"
+
+if [ ! -f ./vmware_inventory.py ]; then
+    echo "DEBUG: Unable to find vmware_inventory.py inside ${contrib_dir}"
+    exit 1
+fi
+
+echo "DEBUG: using ${contrib_dir}"
+
+export ANSIBLE_CONFIG=ansible.cfg
+export vcenter_host="${vcenter_host:-0.0.0.0}"
+export VMWARE_SERVER="${vcenter_host}"
+export VMWARE_USERNAME="${VMWARE_USERNAME:-user}"
+export VMWARE_PASSWORD="${VMWARE_PASSWORD:-pass}"
+
+VMWARE_CONFIG=${contrib_dir}/vmware_inventory.ini
+
+
+trap cleanup INT TERM EXIT
+
+# Remove default inventory config file
+if [ -f "${VMWARE_CONFIG}" ];
+then
+    echo "DEBUG: Creating backup of ${VMWARE_CONFIG}"
+    cp "${VMWARE_CONFIG}" "${VMWARE_CONFIG}.bk"
+fi
+
+cat > "${VMWARE_CONFIG}" <<VMWARE_INI
+[vmware]
+server=${VMWARE_SERVER}
+username=${VMWARE_USERNAME}
+password=${VMWARE_PASSWORD}
+validate_certs=False
+VMWARE_INI
+
+function cleanup {
+    # Revert back to previous one
+    if [ -f "${VMWARE_CONFIG}.bk" ]; then
+        echo "DEBUG: Cleanup ${VMWARE_CONFIG}"
+        mv "${VMWARE_CONFIG}.bk" "${VMWARE_CONFIG}"
+    fi
+}
+
+echo "DEBUG: Using ${vcenter_host} with username ${VMWARE_USERNAME} and password ${VMWARE_PASSWORD}"
+
+echo "Kill all previous instances"
+curl "http://${vcenter_host}:5000/killall" > /dev/null 2>&1
+
+echo "Start new VCSIM server"
+curl "http://${vcenter_host}:5000/spawn?datacenter=1&cluster=1&folder=0" > /dev/null 2>&1
+
+echo "Debugging new instances"
+curl "http://${vcenter_host}:5000/govc_find"
+
+# Change Python interpreter in vmware_inventory
+sed -i 's/ python/ python2/' ./vmware_inventory.py
```

Comment:
https://github.com/ansible/ansible/pull/45995 will eliminate the need for hacking the shebang on the inventory script and will allow collection of code coverage. Creating an inventory wrapper script will take care of things then:

```bash
#!/usr/bin/env bash
python.py "../../../../contrib/inventory/vmware_inventory.py" "$@"
```

Reply:
OK.